label_sync/labels.yaml

```diff
@@ -1554,6 +1554,12 @@ repos:
         target: both
         prowPlugin: label
         addedBy: anyone
+      - color: f6c5d8
+        description: Issues or PRs that need to be closed before the next minor release
```

Comment:
Of CAPI?

Reply:
I'd also remove `minor` and just say the next release

Reply:
all the other labels in that section/file start with "Issues or PRs that..." without mentioning the project since it's in the name of that list, I was going for consistency - what do you think?

Reply:
Changed to "the next CAPI release"